### ISSUE TITLE
Flockingbird scraping behaviour

### ISSUE THREAD

**Issue by playmobilmeister** [Flockingbird/hunter2#18]
A conversation about Flockingbird scraping behaviour has been developing using the #FediBlock hashtag on some Fediverse ActivityPub networks..

Please clarify how you do things. My understanding is that a job listing is searched only for profiles which are already publicly indexed, and that #nobot on the profile bio is not required, but people seem to be thinking otherwise. As someonw who supports your project but is also an admin, may I suggest that you post something fast, or the block will spread quickly and it would be hard to turn things around. 
much respect for the work that you did so far.

Example posts:
[1](https://cloudisland.nz/@aurynn/109792389162591606)
[2](https://aus.social/@ajsadauskas/109719069603063996)
[3](https://mastodon.surazal.net/@zalasur/109791936418409286)
[4](https://poketopia.city/@ignis/109376888164872206)

**Comment by berkes** [comment]
Thanks for the heads-up. The angry mobs already had us banned from a few instances. So maybe we must just let it blow over.

They won't ban any instance, just an account or two that already are closed down. So as an admin, you probably don't have to do anything. 

Reacting doesn't help. We honestly did not do anything wrong. And those folks that went past their anger and actually *talked* to us (rather than slathering and spreading lies behind backs), all of those we showed that there's nothing wrong here.
Facts won't matter, reasonable posts won't matter, an angry mob has made up their minds, based on falsehoods. And there's no convincing them.

The fediverse has a serious bullying problem. Which, ironically, seems to be the reason why many folks want "us" banned in the first place: to avoid bullying. So they bully until people give up, and move to the next victim.

We haven't decided what to do, but we'll probably just shelve the whole project: another fediverse project, tool and idea kicked into the ground by a few angry, ill-informed radicals. This *is* a problem we (we: the fediverse) need to solve, because these "mobs" are doing serious harm. They are directing hate, even threats, at *people* at us, for example. Bullying, dogpiling, and stopping any innovation just because they think they can. While, in the end, it matters nothing: it's not as if these actions stop those that *actually* scrape and index the fediverse: they only kick those into the curb that are open and transparant about what they do, those who try their best to consider everyone and each angle.

**Comment by playmobilmeister** [comment]
I understand your frustration, believe me. as on admin who's easy on the trigger myself, I have seen quite a few unjustified #fediblock content that spreads like wildfire, and when you check the details your opinion may change. 

Now let me share my perspective, as someone who wants your project to live on but is also an active moderator of a community of thousands that expects me to keep an eye on server de-federation.

1. I have maybe 15 min. a day to check #fediblock for new block recommendations, validate them myself (contrary to some others) by going in and looking for clues, and decide whether to do something. This includes unique cases like yours, along with many many **many** more cases of regular micro-blogging servers that are sometimes very easy and sometimes very hard to figure out. Those that are hard to figure out I usually limit but not suspend, if only for the lack to time to investigate further. This leaves out very little time to cases like yours.

2. 90% of block recommendations I eventually agree with and implement myself, and this is the reason why the "mob" rule works. It works because most recommendations are genuine and very easy to decide, and are not bullying. You go in, you see the hate speech or the non-existent moderation or whatever else it is, and it's very obvious . After a while I guess most people start trusting the so called mob because the pattern is strong correlation between them and those other admins. Again, the incentive to go along is very strong because there's very little time to decide.

3. I had ~5 minutes to decide your case and that is because I decided to give you the benefit of the doubt. You claim to be open and transparent, yet in a couple of minutes I found no clear and concise explanation on https://search.flockingbird.social/ and in the repository. maybe there is and I missed it, maybe there isn't. Either way, there's 3,000 people whose profiles I am expected to protect. I see that your project is ~2 years old I am a volunteer and so are 99% of the other admins, most of them do not even know how to go about checking this.

I hope that you do not give up but also that you take the above scenario into consideration.
I hope that you do not let it blow over, because it might not. I would like to find my next job through your platform.
I hope I am not too bold in suggesting that in the main page you add a note for fediverse users and admins that summarizes  those facts, reasonable explanations and falsehoods.
